Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,163 @@
+﻿using System;
+using System.Globalization;
+
+namespace BootCamp.Chapter
+{
+    public class Lesson3
+    {
+        public static void Demo()
+        {
+            const int maxEntries = 2;
+
+            for (int i = 1; i <= maxEntries; i++)
+            {
+                Console.Clear();
+
+                RegisterNewPersonData(i);
+
+                PromptNewPerson(i);
+            }
+        }
+
+        private static string WriteNewLine(int count)
+        {
+            string newLines = Environment.NewLine;
+
+            for (int i = 1; i < count; i++)
+            {
+                newLines += Environment.NewLine;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use a StringBuilder instead.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4884734)